src/makeUrls.ts

```diff
@@ -37,8 +37,8 @@ const makeGoogleCalendarUrl = (event: CalendarEvent) => makeUrl("https://calenda
 
 const makeOutlookCalendarUrl = (event: CalendarEvent) => makeUrl("https://outlook.live.com/owa", {
   rru: "addevent",
-  startdt: makeTime(event.startsAt),
-  enddt: makeTime(event.endsAt),
+  startdt: new Date(event.startsAt).toISOString(),
```

Comment:
We're expecting that `event.startsAt` and `event.endsAt` is an ISO-formatted string here anyway, so let's just pass the values directly through instead of using the `Date` object.

Reply:
right